app/controllers/users/invitations_controller.rb

```diff
@@ -1,13 +1,25 @@
 class Users::InvitationsController < Devise::InvitationsController
-  before_action :authorise_manage_team, only: %i(create new)
   before_action :delete_user_record, if: :user_should_be_cleared?, only: :create
-  before_action :add_organisation_to_params, :validate_invited_user, only: :create
-
+  before_action :add_organisation_to_params, only: :create
```

Comment:
We've lost `before_action validate_invited_user, only: :create`. Where is the user being validated now?

Reply:
Validation is run as part of the invite steps, so part of the base `Devise::InvitiationsController`

Reply:
👍